cmd/accounts.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// accountsCmd represents the accounts command
+var accountsCmd = &cobra.Command{
+	Use:   "accounts",
+	Short: "Exports the account data.",
+	Long:  `Exports historical account data within the specified ledger range to an output file.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		/*
+			Functionality planning:
+			1. Read in start and end ledger numbers
+			2. Provide ledger number to ingestion system and receive data
+			3. Convert data from ingestion into an array of Ledger structs with length equal to the limit
+			4. Write array to output file
+			TODO: Consider having a way to export directly to BigQuery using bigquery library
```

Comment:
Good foresight thinking about the rest of the project. Note that we want to use the Google Cloud Composer (their managed service build above Airflow) for this part. We would likely use an Airflow task for this, since there are custom Airflow operators for BigQuery ([example](https://airflow.apache.org/docs/stable/_api/airflow/contrib/operators/bigquery_operator/index.html)). So, exporting to BigQuery doesn't have to be part of this CLI tool.